### PR TITLE
Introduced Hindu Solar Old calendar.

### PR DESCRIPTION
**Pull request by catull**
Commented out references to Indian Civil, in favour of Hindu Solar Old calendar.

### CODE REVIEW

**Comment by dzucconi** [comment]
Could you provide a bit of context for this change?

**Comment by catull** [comment]
TLDR: Indian Civil calendar is not defined in "Calendrical Calculations", so the test data we have for Indian / Hindu calendars does not fit.

In Calendrical Calculations, there is no Indian Civil calendar.
However, 6 different Hindu calendars are defined:
 - Old Hindu Solar
 - Old Hindu Lunar
 - New Hindu Solar
 - New Hindu Lunar
 - Astronomical Hindu Solar
 - Astronomical Hindu Lunar

These 6 come with test data.
When I tried to apply the test data to the implementation of Indian Civil calendar, nothing ran successfully.  The test data I refer to is in test/spec/data/data4.js.

Since I cannot find a reference to Indian Civil calendar in Calendrical Calculations, I commented it out.
I want to revisit it, come up with test data, too.

But first I want to cover the remaining cases that don't run 100%.
These are the Hindu calendars and the two Persian calendars.

Right now I am introducing the Old Hindu Lunar calendar, all cases except for two are successful.

I am fine to leave the Indian Civil calendar in, but switch off the tests for now.

**Comment by dzucconi** [comment]
I see, that makes sense to me. Thanks for the context!
